src/wp-content/themes/twentytwelve/css/ie.css

```diff
@@ -270,4 +270,4 @@ input[type="password"] {
 .ie7 .rtl .main-navigation li ul ul ul {
 	position: relative;
 	z-index: 1;
-}
\ No newline at end of file
+}
```

Comment:
Restoring the `ie.css` and `html5.js` added a line break at the end of both files. This is not exactly a problem, but I would prefer not to modify those files at all.

Reply:
it seems like the editor added this by default, restored the original versions as it should be, thank you